jsapp/js/components/permissions/transferProjects/transferProjects.component.tsx

```diff
@@ -181,7 +181,7 @@ export default function TransferProjects(props: TransferProjectsProps) {
               label={t('Transfer')}
               isFullWidth
               onClick={toggleModal}
-              color='storm'
+              color='dark-blue'
```

Comment:
I wonder if the `t('Transfer project')` button should be danger too? 

![Screenshot_20240712_142020](https://github.com/user-attachments/assets/0c614764-3781-4789-9bff-b7d39b6bc408)

(or would that be overdoing it)

Reply:
@p2edwards I think the "Transfer Project" button shouldn't be red because it could cause unnecessary anxiety for users and might be misinterpreted as an extremely dangerous action, similar to deleting a form for example. The red color for the warning message already emphasizes caution effectively. By using the blue primary button, it's more consistent with our branding and less alarming, creating a smoother user experience.